public/src/main/java/com/revenuecat/purchases/Offering.kt

```diff
@@ -7,7 +7,7 @@ package com.revenuecat.purchases
 
 import android.os.Parcelable
 import kotlinx.android.parcel.IgnoredOnParcel
```

Comment:
I believe this one also needs to be changed like I did on https://github.com/RevenueCat/purchases-android/pull/474/files#diff-869b813b31b540b6a7fbf9b7fa2d5bffe1011dcda27ce036be533907e9a5b47fR9, but I am not sure

Reply:
Good catch! I would have thought this wouldn't compile... Will fix it on a followup PR. Thanks!